.circleci/config.yml

```diff
@@ -0,0 +1,216 @@
+    environment:
+      - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
+      - SCONSFLAGS: "-j3"
+    steps:
+      - checkout
+      - run:
+          name: Examples STM32F4 Series (except STM32F4 Discovery Board)
+          command: |
+            scons check=examples examples=stm32f429_discovery,stm32f469_discovery,nucleo_f401re,nucleo_f411re,nucleo_f429zi
+
+  "ex-stm32f7":
+    docker:
+      - image: stronglytyped/xpcc-build-arm:latest
+    environment:
+      - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
+      - SCONSFLAGS: "-j3"
+    steps:
+      - checkout
+      - run:
+          name: Examples STM32F7 Series
+          command: |
+            scons check=examples examples=stm32f746g_discovery
+
+  "ex-stm32l4":
+    docker:
+      - image: stronglytyped/xpcc-build-arm:latest
+    environment:
+      - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
+      - SCONSFLAGS: "-j3"
+    steps:
+      - checkout
+      - run:
+          name: Examples STM32L4 Series
+          command: |
+            scons check=examples examples=stm32l476_discovery,nucleo_l476rg
+
+  "ex-avr":
+    docker:
+      - image: stronglytyped/xpcc-build-arm:latest
+    environment:
+      - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
+      - SCONSFLAGS: "-j3"
+    steps:
+      - checkout
+      - run:
+          name: Examples ATmega Series
+          command: |
+            scons check=examples examples=avr,arduino_uno
+
+  "ex-lpcxpresso":
+    docker:
+      - image: stronglytyped/xpcc-build-arm:latest
+    environment:
+      - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
+      - SCONSFLAGS: "-j3"
+    steps:
+      - checkout
+      - run:
+          name: Examples LPCxpresso Board
+          command: |
+            scons check=examples examples=lpcxpresso
+
+  "ex-linux":
+    docker:
+      - image: stronglytyped/xpcc-build-arm:latest
+    environment:
+      - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
+      - SCONSFLAGS: "-j3"
+    steps:
+      - checkout
+      - run:
+          name: Examples Linux
+          command: |
+            scons check=examples examples=linux,zmq
+
+  "devices":
+    docker:
+      - image: stronglytyped/xpcc-build-arm:latest
+    environment:
+      - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
+      - SCONSFLAGS: "-j3"
+    steps:
+      - checkout
+      - run:
+          name: Check devices
+          command: |
+            scons check=devices
+
+workflows:
+  version: 2
+  build:
+    jobs:
+      # Start longest running jobs first
+      - "ex-stm32f4_discovery"
+      - "ex-stm32f4"
+      - "devices"
+      - "ut-hosted"
+      - "ut-stm32"
+      - "ut-atmega"
+      - "ex-stm32f0"
+      - "ex-stm32f1"
+      - "ex-stm32f3"
+      - "ex-stm32f7"
+      - "ex-stm32l4"
+      - "ex-lpcxpresso"
+      - "ex-linux"
+      - "ex-avr"
```

Comment:
I like this split very much more than the monoblock from travis, makes it much easier to see the failure on GitHub.

Reply:
I would think this is a feature of CircleCI. I have no idea how to split Travis into multiple parts.